rotkehlchen/tests/api/test_ens.py

```diff
@@ -81,13 +85,13 @@ def test_reverse_ens(rotkehlchen_api_server):
         status_code=HTTPStatus.BAD_REQUEST,
     )
 
-    timestamps_before_request = _get_timestamps(db, all_addrs)
     requests.post(
         api_url_for(
             rotkehlchen_api_server,
             'reverseensresource',
         ),
         json={'ethereum_addresses': all_addrs, 'ignore_cache': True},
     )
-    timestamps_after_request = _get_timestamps(db, all_addrs)
-    assert all(map(lambda t_pair: t_pair[0] < t_pair[1], zip(timestamps_before_request, timestamps_after_request)))  # noqa: E501
+    db_changes_after = db_conn.total_changes
+    # Check that we have 5 updates because we have 5 rows in ens_mappings table
+    assert db_changes_after == 5 + db_changes_before
```

Comment:
nitpick: Unnecessary newline. Also comment would be assert message

```suggestion
    # Check that we have 5 updates because we have 5 rows in ens_mappings table
    assert db_changes_after == 5 + db_changes_before, 'should have made 5 DB changes'
```